examples/client/Locomotion/src/context/user/index.tsx

```diff
@@ -106,6 +114,15 @@ const UserContextProvider = ({ children }: { children: any }) => {
     }
   };
 
+  const onEmailVert = async (code: string) => {
+    const vertResponse = await emailVerify({
+      email: user?.email,
+      code,
+    });
+
+    return vertResponse.status === 'OK';
```

Comment:
Let's work with status codes

Reply:
Using the status code for this does not make sense

Reply:
This is not client error like bad password is not 400

Reply:
firebase do it as well